l10n_do_accounting/i18n/es_DO.po

```diff
@@ -511,8 +511,8 @@ msgstr "Nombre mostrado"
 #. module: l10n_do_accounting
 #: code:addons/l10n_do_accounting/models/l10n_latam_document_type.py:0
 #, python-format
-msgid "Document number %s doesn't match expected pattern"
-msgstr "El NCF %s no coincide con la estructura esperada"
+msgid "Document number %s doesn't fit with the expected structure"
```

Comment:
Sugerencia de términos: 

The structure of Document number is invalid.
La estructura del Número de Documento es invalida.